tests/framework/types.go

```diff
@@ -39,22 +39,22 @@ type VirtualMachine struct {
 
 func (vm VirtualMachine) Create() (string, string, error) {
 	args := []string{"create", "-f", vm.Manifest}
-	return ktests.RunCommandWithNS(vm.Namespace, ktests.KubeVirtOcPath, args...)
+	return ktests.RunCommandWithNS(vm.Namespace, "oc", args...)
```

Comment:
Why did you change ktests.KubeVirtOcPath to "oc", if from some reason the command will change it's easier to change it in one place 'ktests.KubeVirtOcPath'.

Reply:
created **oc** and **virtctl** as constants